test/unit/core.js

```diff
@@ -1709,3 +1709,45 @@ QUnit.test( "Iterability of jQuery objects (gh-1693)", function( assert ) {
 		assert.ok( true, "The browser doesn't support Symbols" );
 	}
 } );
+
+QUnit[ jQuery.Deferred ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
```

Comment:
Yes, I did a pass with a custom build replacing ready with ready-no-deferred on the whole suite. Guards like this are necessary if the test is not applicable to ready-no-deferred.